cmd/subnet-utils.go

```diff
@@ -807,7 +812,15 @@ func initSubnetConnectivity(ctx *cli.Context, aliasedURL string, forUpload bool,
 }
 
 func validateSubnetFlags(ctx *cli.Context, forUpload bool) error {
-	if !globalAirgapped {
+	supportsAirgap := false
```

Comment:
This may work, ctx.Bool("airgap") ?

Reply:
ctx.IsSet("airgap") would also work

Reply:
> This may work, ctx.Bool("airgap") ?

I think this will give the value of the `--airgap` flag

> ctx.IsSet("airgap") would also work

and this will tell if the `--airgap` flag was set

But what I'm trying to do is to check if this flag is even supported by the command being executed. The subsequent validation (that says `--json is applicable only when --airgap is also passed`) should be done only for commands that support the `--airgap` flag. 

The command I'm adding (`mc support upload`) doesn't support `--airgap` flag but supports `--json`

Reply:
--json must be present for even successful operations towards SUBNET - not just in --airgap mode.

Reply:
@harshavardhana sure, I can change it that way. In that case, say for example in case of `mc support diag`, `--json` will print the diagnostics report to screen in case of `--airgap`, and something like `{"status": "success"}` in non-airgapped execution. Does that sound ok?

Reply:
@harshavardhana I have sent https://github.com/minio/mc/pull/4859 for removing this validation (and other related changes)

Reply:
this function has been removed now.